internal/cluster/clusters.go

```diff
@@ -90,8 +103,9 @@ func (e *clusterModelNotFoundError) Error() string {
 
 func (e *clusterModelNotFoundError) Context() []interface{} {
 	return []interface{}{
-		"cluster", e.cluster,
-		"organization", e.organizationID,
+		"clusterID", e.cluster.ID,
```

Comment:
don't we pass these fields to the API? please update openapi description if we do

Reply:
I see no relevant use of `cluster` or `organization` as property name in the API description, nor does `ClusterNotFound` refer to them, so I don't think they're passed anywhere.